src/composables/useConversationInfo.ts

```diff
@@ -116,6 +120,31 @@ export function useConversationInfo({
 			}
 		}
 
+		// This is for event conversations where no messages from participants are shown
+		const startTime = getEventTimeRange(item.value).start
+		if (item.value.objectType === CONVERSATION.OBJECT_TYPE.EVENT
+			&& startTime && startTime > Date.now()) {
+			const store = useStore()
+			const hasHumanMessage = store.getters.messagesList(item.value.token).some((message: ChatMessage) => {
```

Comment:
I'd still use `[item.value.lastMessage]` as a fallback here, in case you never opened a chat yet, but someone else is already writing there
<img width="185" alt="image" src="https://github.com/user-attachments/assets/7d2174bc-3f5a-483d-9824-6f85b5f9ac1f" />

Reply:
Was it resolved?